src/components/CountriesLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from "react";
+import { CountryLink } from "@components/CountryLink";
+import { getCountryName } from "@helpers/getCountryFields";
+import { isSystemGeo } from "@utils/isSystemGeo";
+import { TGeography } from "@types";
+
+type TCountriesLink = {
+  geographies: string[];
+  countries: TGeography[];
+};
+
+export const CountriesLink: FC<TCountriesLink> = ({ geographies, countries }) => (
+  <>
+    {geographies?.map(
+      (geography, index) =>
+        !isSystemGeo(geography) && (
+          <span key={index} className="flex gap-1">
```

Comment:
Use the geo as the key, index as key should only be for situations where we cannot be sure the values in the collection will be distinct.